spec.bs

```diff
@@ -68,6 +70,15 @@ spec: webdriver; urlPrefix: https://w3c.github.io/webdriver/
         text: WebDriver error code; url: dfn-error-code
 </pre>
 
+<pre class="biblio">
+{
+    "webauthn-conditional-ui": {
+        "title": "WebAuthn Conditional UI Proposal",
+        "href": "http://www.example.com/foo-bar/"
```

Comment:
Do you mean this URI?
 https://github.com/w3c/webauthn/pull/1576

Reply:
Hah, good catch! I said to myself I'll fix that later... and never did. Fixed now.